sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormTrainingAsyncClientTest.java

```diff
@@ -54,6 +61,25 @@ private FormTrainingAsyncClient getFormTrainingAsyncClient(HttpClient httpClient
         return getFormTrainingClientBuilder(httpClient, serviceVersion).buildAsyncClient();
     }
 
+    /**
+     * Verifies the form recognizer async client is valid.
+     */
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.formrecognizer.TestUtils#getTestParameters")
+    void getFormRecognizerClientAndValidate(HttpClient httpClient, FormRecognizerServiceVersion serviceVersion) {
+        FormRecognizerAsyncClient formRecognizerClient = getFormTrainingAsyncClient(httpClient, serviceVersion)
+            .getFormRecognizerAsyncClient();
+        blankPdfDataRunner(data -> {
+            SyncPoller<OperationResult, List<RecognizedReceipt>> syncPoller =
+                formRecognizerClient.beginRecognizeReceipts(
+                    new RecognizeOptions(toFluxByteBuffer(data), BLANK_FORM_FILE_LENGTH)
+                        .setFormContentType(FormContentType.APPLICATION_PDF).setIncludeTextContent(false))
+                    .getSyncPoller();
+            syncPoller.waitForCompletion();
+            validateBlankPdfResultData(syncPoller.getFinalResult());
```

Comment:
validateBlankPdfResultData() is this method is only specific to Receipt result lets update it to `validateBlankReceiptResult()` ?

Reply:
Receipt and Custom both have the Blank PDF test. So I think we should be restricted to `Receipt`